nixfromnpm.cabal

```diff
@@ -32,20 +32,25 @@ source-repository head
   location: git://github.com/adnelson/nixfromnpm.git
 
 library
+  default-language: Haskell2010
   hs-source-dirs:      src
   exposed-modules:     NixFromNpm
-  other-extensions:    FlexibleContexts
-                     , FlexibleInstances
-                     , LambdaCase
+                     , NixFromNpm.Common
+                     , NixFromNpm.Options
+                     , NixFromNpm.Conversion.ToDisk
+                     , NixFromNpm.Git.Types
+                     , NixFromNpm.Npm.Version
+                     , NixFromNpm.Merge
+  other-modules:       Filesystem.Path.Wrappers
```

Comment:
My only issue with using `other-modules` is that it can be frustrating if I need to access something inside one of these modules directly that isn't explicitly exported by others.

One way that I've seen people successfully solve this problem of maintaining a private interface without restricting user's ability to import something directly from a private module is to export all modules but qualify modules that are part of the "private" namespace with `Internal` to indicate to the user that they should know what they are doing if they use it.

e.g. `NixFromNpm.Internal.Npm.Types`

Reply:
What I did as a first pass was just put all of the modules in the package into `other-modules` and move them into `exposed-modules` until all of the ones used by the executable and tests were exposed. I wouldn't have a problem just exposing everything, tbh. Especially for a package like this, it doesn't seem necessary to hide too many details.

Reply:
Oh I understand. Sounds good to me.